src/PKSim.Core/Commands/SetRelativeExpressionAndNormalizeCommand.cs

```diff
@@ -1,18 +1,18 @@
-using PKSim.Core.Model;
 using OSPSuite.Core.Domain;
+using PKSim.Core.Model;
 
 namespace PKSim.Core.Commands
 {
    public class SetRelativeExpressionAndNormalizeCommand : PKSimMacroCommand
    {
       private readonly double _newValue;
       private IndividualMolecule _molecule;
-      private IParameter _relExpParameter;
+      private IParameter _relativeExpressionParameter;
 
-      public SetRelativeExpressionAndNormalizeCommand(IndividualMolecule molecule, string proteinContainerName, double value)
+      public SetRelativeExpressionAndNormalizeCommand(IndividualMolecule molecule, IParameter relativeExpressionParameter, double value)
       {
          _molecule = molecule;
-         _relExpParameter = molecule.GetRelativeExpressionParameterFor(proteinContainerName);
```

Comment:
Not sure why we were passing a string to retrieve the parmeter from the container. What about we pass the parameter instead ?